### ISSUE TITLE
About UCF accuracy

### ISSUE THREAD

**Issue by 420646016** [naviocean/pseudo-3d-pytorch#5]
Excuse me. I would like to ask if the accuracy 81.6% you gave on UCF101 dataset is the top1 or the top5?
Since I only have one piece of 1070Ti, my batchsize can only be set to 6.
I trained from scratch according to the steps you gave, and found that the loss cannot converge. 
On the test dataset, the accuracy of top1 is 42%  and that of top5 is 68%
Do you think the batchsize might have influenced the final result? Thanks for your help.

**Comment by naviocean** [comment]
Its top1 accuracy.
